Manifests/ManagedPreferencesApplications/us.zoom.config.plist

```diff
@@ -284,10 +393,12 @@
 			<string>SSO URL</string>
 			<key>pfm_type</key>
 			<string>string</string>
+			<key>pfm_value_placeholder</key>
+			<string>zoom_subdomain</string>
```

Comment:
I didn't immediately understand "zoom_subdomain". Maybe let's change it to "Example: 'myorg' when using a login URL of myorg.zoom.us"?

I think we have precedence in the repository of placeholders starting with "Example:".